src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -154,6 +154,21 @@ export const shareModalStrings = {
       defaultMessage:
         'One or more panels on this dashboard have changed. Before you generate a snapshot, save the dashboard.',
     }),
+  getDraftSharePanelChangesWarning: () =>
```

Comment:
I think the wording here should be for `getDraftShareWarning` and we can get rid of `getDraftSharePanelChangesWarning`? cc @florent-leborgne

Reply:
Should we be communicating that the link might not work in a scenario where we don't need to, we've ascertained that the issue with the link possibly not working only happens when we have changes in the panels?

Reply:
Bumping this, since I know some users are pretty desperate to get this resolved and so I think pushing forward with this review would be appreciated 🙇

Reply:
Hey hey... apologies for the delay, I was OOO last week. The copy for embed has been modified to account both control changes and dashboard panel changes.

Reply:
Sorry, @eokoneyo. One last thing 🙇 Can we change the text in embed mode to be consistent?

![image](https://github.com/user-attachments/assets/2bd05dee-03cc-4f68-9059-afc7ec37e698)

Right now it says "Consider saving your dashboard before generating the link" on the embed tab - can we change it to something like "Consider saving your dashboard before generating the embed code"?

Reply:
Thanks for the catch. I attempted to reuse the text without making a distinction, this has been corrected.